swirl/connectors/connector.py

```diff
@@ -329,7 +329,7 @@ def process_results(self):
             logger.info(f"{self}: invoking processor: process results {processor}")
             last_results = copy.deepcopy(self.results)
             try:
-                proc = eval(processor, {"processor": processor, "__builtins__": None}, SWIRL_OBJECT_DICT)(self.results, self.provider, self.query_string_to_provider)
+                proc = eval(processor, {"processor": processor, "__builtins__": None}, SWIRL_OBJECT_DICT)(self.results, self.provider, self.query_string_to_provider, self.provider.tags)
                 modified = proc.process()
                 self.results = proc.get_results()
```

Comment:
We are already passing the provider, why do we have to pass the 'tags' separately?  It's not desirable  as it is a breaking change to the processor interface.